docs/source/notes/amp_examples.rst

```diff
@@ -0,0 +1,210 @@
+.. _amp-examples:
+
+Automatic Mixed Precision Examples
+==================================
+
+When following the examples below, you don't need to call ``.half()`` on your model(s) or data.
+In fact, you shouldn't.  Model weights should remain FP32.
+
+You also don't need to retune any hyperparameters.
+
+.. contents:: :local:
+
+.. _autocasting-examples:
+
+Autocasting Examples
+^^^^^^^^^^^^^^^^^^^^
+
+Under construction...
+
+.. _gradient-scaling-examples:
+
+Gradient Scaling Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The code snippets below demonstrate recommended use of :class:`torch.cuda.amp.AmpScaler`.
+
+.. currentmodule:: torch.cuda.amp
+
+Typical use (1 loss, 1 optimizer)
+---------------------------------
+
```

Comment:
Need to elaborate on what the snippet does. Describe the new functionality, when you should call it, and why.